cpp/include/cuvs/neighbors/cagra.hpp

```diff
@@ -1066,7 +1068,8 @@ void search(raft::resources const& res,
             raft::device_matrix_view<uint32_t, int64_t, raft::row_major> neighbors,
             raft::device_matrix_view<float, int64_t, raft::row_major> distances,
             const cuvs::neighbors::filtering::base_filter& sample_filter =
-              cuvs::neighbors::filtering::none_sample_filter{});
+              cuvs::neighbors::filtering::none_sample_filter{},
+            double threshold_to_bf = 0.9f);
```

Comment:
Please remove this argument, and place it into the `search_params` structure instead. The search API should stay the same.

Reply:
Done